example/nsfw_demo/src/App.js

```diff
@@ -138,16 +138,22 @@ class App extends Component {
   }
 
   detectWebcam = async () => {
-    const video = document.querySelectorAll('.captureCam')
-    const predictions = await this.state.model.classify(video[0])
-    let droppedImageStyle = this.detectBlurStatus(predictions[0].className)
-    this.setState({
-      message: `Identified as ${predictions[0].className}`,
-      predictions,
-      droppedImageStyle
-    })
+      // await this.sleep(100)
+
+      const video = document.querySelectorAll('.captureCam')
+
+      if (video[0]){
```

Comment:
if (video[0]){

- value is undefined at the start.... it crashes if taken further